apps/web-giddh/src/app/vouchers/create/create.component.ts

```diff
@@ -3830,7 +3846,7 @@ export class VoucherCreateComponent implements OnInit, OnDestroy, AfterViewInit
                     }
                 });
             } else {
-                this.voucherService.generateVoucher(invoiceForm.account.uniqueName, invoiceForm).pipe(takeUntil(this.destroyed$)).subscribe(response => {
+                this.voucherService.generateVoucher(accountUniqueName, invoiceForm).pipe(takeUntil(this.destroyed$)).subscribe(response => {
```

Comment:
Whats the use? You already assigned above

Reply:
This is used for cash invoices where we use the deposit's unique name in the URL instead of the account's unique name.

Reply:
Are to 3832 line pe account unique name assign kiya hena, to isko change karne ka kya use he?